tests/mmm/test_base.py

```diff
@@ -210,7 +210,11 @@ def toy_preprocessing(self, data):
             ("plot_grouped_contribution_breakdown_over_time", {}),
             (
                 "plot_grouped_contribution_breakdown_over_time",
-                {"stack_groups": {"controls": ["control_1"]}},
+                {
```

Comment:
Not for this PR, but I wonder if we should test the actual expected plots with something like https://matplotlib.org/stable/api/testing_api.html#module-matplotlib.testing

At least we will know if they accidentally change. Not super convinced.

Reply:
wow! I did not know about this! I'll take a look and create an issue